tasks/47_string-expansion/string-expansion.js

```diff
@@ -0,0 +1,12 @@
+function stringExpansion(str) {
+  let n = 1;
+  return Array.from(str).reduce((sum, item, index) => {
```

Comment:
Returning chaining functions with a bunch of complex logic is a bad practice. Please separate it in different lines